pkgs/apps/gamess-us/default.nix

```diff
@@ -68,8 +70,11 @@ stdenv.mkDerivation rec {
       # Prepare the rungms script -> replace references to @version@ and @out@
       export mpiname=${mpiname}
       export mpiroot=${mpiroot}
+      export target=${target}
       substituteAllInPlace rungms
 
+      head -n 200 rungms
```

Comment:
Does this line have purpose, or was this for debugging purposes?

Reply:
Unfortunately only debugging. Forgot about it :/